### PR TITLE
Dark Mode Beta

### PR DESCRIPTION
**Pull request by nyinyi979**
I have tested out dark mode in local version. The details of the hsl variables in global.css has been altered to make the dark mode work properly. I will host this version on Vercel to visualize the changes a bit more:) . I would love to be part of the contributions!

### CODE REVIEW

**Comment by amm834** [comment]
Screenshot 2023-11-01 at 8 56 18 PM
In above image:
- Border is too bold
- Bold text and so on

File: apps/myanmar_calendar/src/components/modals/DayDialog/DayDialog.tsx [hunk 1]
```diff
@@ -64,29 +64,29 @@ const DayDialog = ({ onClose, selectedDay }: DayDialogProps) => {
...
         enter={`ease-out ${enterMobileMode ? "duration-200" : " duration-200"}`}
         leave={`ease-in ${enterMobileMode ? "duration-200" : " duration-150"}`}>
-        <Dialog.Panel className="fixed inset-0 mx-auto mt-auto sm2:my-auto w-full sm2:max-w-[27rem] h-[calc(100%-5rem)] sm2:h-[90%] sm2:max-h-[33rem] transform overflow-hidden rounded-tr-2xl rounded-tl-2xl sm2:rounded-[0.5rem] bg-white text-left align-middle transition-all flex flex-col sm2:border  sm2:border-gray-200 shadow-model">
+        <Dialog.Panel className="fixed inset-0 mx-auto mt-auto sm2:my-auto w-full sm2:max-w-[27rem] h-[calc(100%-5rem)] sm2:h-[90%] sm2:max-h-[33rem] transform overflow-hidden rounded-tr-2xl rounded-tl-2xl sm2:rounded-[0.5rem] bg-cgray-0 text-left align-middle transition-all flex flex-col sm2:border  sm2:border-cgray-200 shadow-model">
           {/* ------ Header ------ */}
-          <div className="h-[3.25rem] sm2:h-[3rem] flex items-center justify-between px-5 sm2:px-3 bg-gray-50 border-b">
+          <div className="h-[3.25rem] sm2:h-[3rem] flex items-center justify-between px-5 sm2:px-3 bg-cgray-50 border-b">
```
Comment (amm834) [inline apps/myanmar_calendar/src/components/modals/DayDialog/DayDialog.tsx:69]: Convention like cgray did not give us any extra understanding could you please improve naming convention?

Comment (nyinyi979) [reply apps/myanmar_calendar/src/components/modals/DayDialog/DayDialog.tsx:69]: I think I will go with just gray, maybe that might be sound a bit better, i guess. I meant ColorGray but you are right, it is a bit misleading. I will go with gray and try to fix the boldness!

Comment (amm834) [reply apps/myanmar_calendar/src/components/modals/DayDialog/DayDialog.tsx:69]: Hey thank you for that.
Please fix and updat.

**Comment by nyinyi979** [comment]
Hello there! Before creating a pull request again, I would love to confirm the new UI look
In the month mode: 
NEW UI: 
image
Dark UI: 
image
CURRENT UI:
image

In the week mode: 
NEW UI: 
image
DARK UI: 
image
CURRENT UI: 
image

In the year mode: 
NEW UI:
image
DARK UI:
image
CURRENT UI:
image

1. The lag in NavBar and SideBar is fixed:) 
2. Have changed the variable to gray instead!
3. But I can't do the dark mode without having to create new variables in both :root and .dark ( When I actually look at the original variables, they are almost duplicated values ) . Sorry for that! 
4. In the current version, there are so much of gray-number values used. That's why I created gray variables to utilize them in light mode as normally and reverse in dark mode! 
5. I hope it turns out as you requested. If it is not, I am sorry:(

**Comment by amm834** [comment]
Look cool however shadow of the modal take too much.
Screenshot 2023-11-01 at 10 58 29 PM

And also the Current day should be white text over the black one? Red and Black is not a good combination in my opinion. And also could you pls fix it to be a rounded one?

Screenshot 2023-11-01 at 10 59 22 PM

**Comment by nyinyi979** [comment]
Alright sir, I will fix those in the morning :) 
You can leave more comments so I can work on them tomorrow!

**Comment by YakuBrangJa** [comment]
> Alright sir, I will fix those in the morning :) You can leave more comments so I can work on them tomorrow!


Welcome to the team!

I noticed that in `DarkMode.tsx`, you used a div as a button. To maintain a consistent UI throughout the app, please consider using existing components when possible. In this case, you can utilize the `Button` component from ui/Button.tsx and set its props to variant="ghost" and size="icon". Alternatively, you can apply your custom className.

If you find it necessary to create a new component, we encourage you to explore the components available in [shadcn](https://ui.shadcn.com/).

Keep up the great work 😉.

**Comment by YakuBrangJa** [comment]
> Look cool however shadow of the modal take too much. Screenshot 2023-11-01 at 10 58 29 PM
> 
> And also the Current day should be white text over the black one? Red and Black is not a good combination in my opinion. And also could you pls fix it to be a rounded one?
> 
> Screenshot 2023-11-01 at 10 59 22 PM

I have already fixed the rounded one for active day. If you haven't done it yet, you can leave it.

**Comment by nyinyi979** [comment]
I have fixed the box-shadow, It looks like this:
image
I also used the button variant ghost. 
And the black text colour is fixed: 
image
image
image

**Review by amm834 — APPROVED** [review]

**Pull request merged**